odc/stats/io.py

```diff
@@ -178,14 +179,15 @@ def verify_s3_credentials(self, test_uri: Optional[str] = None) -> bool:
         assert rr.path == test_uri
         return rr.error is None
 
+    # pylint: disable=invalid-name
```

Comment:
I think I'm with pylint on this.  The function argument should be called `content_type`, not `ContentType`.

Reply:
I'm with pylint too. Only not sure if this one is special on purpose.